templates/vue/README.md

```diff
@@ -33,3 +33,37 @@ You need to adjust your git config so the automatic linting process works correc
 ```
 git config core.hooksPath .githooks
 ```
+
+## Testing Setup
+
+The testing framework that we use, Cypress, needs to be configured with some environment variables. In the `cypress.json` file, you should see something like:
+
+```json
+{
+    "env": {
+        "DEV_URL": "http://localhost:8888",
+        "TEST_TAPESTRY_NAME": "testing",
+        "USER_TAPESTRY_NAME": "testing-user",
```

Comment:
Can we remove these? Just name them whatever - the user doesn't care about the name of the tapestries used in testing (right?)

Reply:
By the user do you mean the developers? The `TEST_TAPESTRY_NAME` doesn't matter, but the `USER_TAPESTRY_NAME` matters because it should refer to a test tapestry that was (previously) manually created.